shell/store/type-map.js

```diff
@@ -1010,9 +1010,15 @@ export const getters = {
           formatter = 'Number';
         }
 
+        let colName = col.name;
+
+        if (colName.includes(' ')) {
+          colName = col.name.split(' ').map(word => word.charAt(0).toUpperCase() + word.substring(1) ).join('');
```

Comment:
I'm really intrigued the cases this covers.

This could also be simplified to avoid using the `let`
```
const colName = colName.includes(' ') ?
  col.name.split(' ').map(word => word.charAt(0).toUpperCase() + word.substring(1) ).join('')  :
  col.name: 
```